src/new/wallet.js

```diff
@@ -1749,27 +1857,29 @@ class HathorWallet extends EventEmitter {
     return utxos;
   }
 
+  /**
+   * @typedef {Object} MintTokensOptions
+   * @property {string} [address] destination address of the minted token
+   *                                   (if not sent we choose the next available address to use)
+   * @property {string} [changeAddress] address of the change output
+   *                                   (if not sent we choose the next available address to use)
+   * @property {boolean} [startMiningTx=true] boolean to trigger start mining (default true)
+   * @property {boolean} [createAnotherMint] boolean to create another mint authority or not for the wallet
+   * @property {string} [mintAuthorityAddress] address to send the new mint authority created
+   * @property {boolean} [allowExternalMintAuthorityAddress=false] allow the mint authority address to be from another wallet
+   * @property {boolean} [unshiftData] whether to unshift the data script output
+   * @property {string[]|null} [data=null] list of data strings using utf8 encoding to add each as a data script output
+   * @property {string} [pinCode] pin to decrypt xpriv information.
+   */
```

Comment:
I think it's missing the `signTx` option, isn't it?

Reply:
Yes, added